src/asound/bluealsa-ctl.c

```diff
@@ -938,10 +1043,17 @@ SND_CTL_PLUGIN_DEFINE_FUNC(bluealsa) {
 	snd_config_iterator_t i, next;
 	DBusError err = DBUS_ERROR_INIT;
 	const char *service = BLUEALSA_SERVICE;
+	const char *device_str = NULL;
+	bdaddr_t device;
```

Comment:
Name these variables in the same way as in the `bluealsa-pcm.c`: `device` and `ba_addr`. Also you can use inline initialization:
```c
baaddr_t ba_addr = *BDADDR_ALL;
```